### PR TITLE
#162949206 Setup and deploy base application to Heroku

### PR DESCRIPTION
**Pull request by kwanj-k**
#### What does this PR do?

Setup and deploy application to Heroku

#### Description of Task to be completed?

* Deploy base to production on Heroku
* Set up Postgres on Heroku
* Configure the staging pipeline
* Update settings and other deployments settings
* Update doc string links to point to Django 2.1 documentation

#### How should this be manually tested?

1. Clone the repo: `git clone https://github.com/andela/ah-the-unsullied.git`
2. Cd into the cloned repo 
3. Git pull branch ch-deploy-to-heroku-162949206
4. Set up a virtual environment and install requirements:
      ```
            python3 -m venv venv
            source venv/bin/activate
            pip3 install -r requirements.txt

     ```
5. Set up a .env file in the same level as the venv file and add the following:
      ```
            source venv/bin/activate
            export SECRET_KEY='yoursecretkey'
             export DATABASE_URL='postgres://yourdatabasename'

      ```
6. Then run: 
     ``` 
            source .env 
            ./manage.py migrate
            ./manage.py runserver
      ```
7. To test locally, create a superuser and visit the route provided by the server to test admin site:
      ```
             ./manage.py createsuperuser - to createsuperuser

      ```
8. To test the Live version visit:
  * [Base](https://ah-the-unsullied.herokuapp.com/admin/)
  * [Staging](https://ah-the-unsullied-staging.herokuapp.com/admin/)

#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[162949206](#162949206)
#### Screenshots (if appropriate)
None
#### Questions:

### CODE REVIEW

**Review by MbugwaSami — COMMENTED** [review]
Good work kwanj

**Review by MbugwaSami — DISMISSED** [review]

File: .gitignore [hunk 1]
```diff
@@ -6,6 +6,10 @@ __pycache__/
 # C extensions
 *.so
 
+#vscode files
+*.vscode
+
+
```
Comment (indungu) [inline .gitignore]: A single blank line is enough, please remove extra whitespace.

**Review by indungu — CHANGES_REQUESTED** [review]

**Review by MbugwaSami — COMMENTED** [review]
LGTM

**Review by MbugwaSami — DISMISSED** [review]

**Review by Allan690 — CHANGES_REQUESTED** [review]
Good work, but kindly shorten the title of your commit message
screenshot 2019-01-10 at 12 37 37

**Review by Allan690 — DISMISSED** [review]

**Review by indungu — CHANGES_REQUESTED** [review]
Hey @kwanj-k , nice work... I still have an issue with your commit message header though.
The scope of the task can be better titled as just `deployment` rather than `Deploy and setup pipelines`, yes?
Also, please adhere strictly to the [commit message conventions](https://github.com/andela/engineering-playbook/tree/master/5.%20Developing/Conventions#commit-message) I shared, the [commit message header]() should ideally be in small caps.

**Review by indungu — APPROVED** [review]

**Review by nesh-dev — APPROVED** [review]

**Review by Allan690 — APPROVED** [review]
I think it looks better now that you implemented the feedback

**Pull request merged**